### PR TITLE
added: Badges

### PR DESCRIPTION
**Pull request by rdshinde**
Added badges to avatars as well as icons.
Screenshot 2022-01-31 at 4 50 42 PM
Screenshot 2022-01-31 at 4 51 17 PM

### CODE REVIEW

**Review by sanjay-xdr — APPROVED** [review]
everything looks good

File: components/badges/badge.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/* General Badge Class */
+.badge {
+  align-items: center;
+  display: flex;
+  font-size: 0.5rem;
```
Comment (shilpe26) [inline components/badges/badge.css:5]: use px unit when the value is less than 1rem

**Review by shilpe26 — COMMENTED** [review]
The Rest looks good.👍🏻

File: components/badges/badge.css (hunk 1 above)
Comment (rdshinde) [reply components/badges/badge.css:5]: I have used relative unit to make it responsive for all kind of users with different font sizes.

**Pull request merged**